xen-ocaml/build.sh

```diff
@@ -26,10 +34,11 @@ PWD=`pwd`
 CFLAGS="-Wall -Wno-attributes ${ARCH_CFLAGS} ${EXTRA_CFLAGS} ${CI_CFLAGS} -DSYS_xen -USYS_linux \
   -fno-builtin-fprintf -DHAS_UNISTD \
   $(pkg-config --cflags $PKG_CONFIG_DEPS) \
-  -I `opam config var prefix`/include/mirage-xen/include"
+  "
```

Comment:
It seems OK to remove this, as it builds fine without it, and mirage-xen depends on xen-ocaml anyway, not the other way around. So I believe this was never doing anything useful in the first place. But I could be confused.